di_website/vacancies/models.py

```diff
@@ -97,33 +97,39 @@ def get_context(self, request):
             data = request.POST.copy()
             email_address = data.get('email_address', None)
             email['value'] = email_address
+            gdpr = data.get('gdpr', None)
+
             try:
-                if email_address:
-                    validate_email(email_address)
-
-                    monitored_departments = []
-                    for department in all_departments:
-                        if data.get(department.slug, None) == 'on':
-                            monitored_departments.append(department)
-
-                    form['alert_message'] = 'You have signed up for vacancy alerts for '
-                    if len(monitored_departments):
-                        first = True
-                        for department in monitored_departments:
-                            self.create_subscription(email.value, 'jobs', department)
-                            if first:
-                                form['alert_message'] = form.get('alert_message') + ' ' + department.name
-                                first = False
-                            else:
-                                form['alert_message'] = form.get('alert_message') + ', ' + department.name
-                    else:
-                        for department in all_departments:
-                            self.create_subscription(email.value, 'jobs', department)
-                        form['alert_message'] = ' all departments'
+                if gdpr:
```

Comment:
I think this can even be simpler, @davidebukali when you changed the form action to the Mail Chimp website, the data is no longer POSTed back to the model, so there's no validation we can do on our end. It's all validated via Mail Chimp's JS.

Reply:
The subscription code from users model seems unnecessary as well because it was being used in vacancies to subscribe users; Let me remove it